tests/datatype/tests_Decimal.cpp

```diff
@@ -0,0 +1,68 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+
+#include <doctest/doctest.h>
+#include <rdf4cpp/rdf.hpp>
+
+using namespace rdf4cpp::rdf;
+
+TEST_CASE("Datatype Decimal") {
```

Comment:
You need to write test cases that check how we handle trailing zeros in the decimal part. The standard says that "1.0" and "1.00" are not distinct.